ArmPkg/Drivers/SmmuDxe/SmmuV3.h

```diff
@@ -0,0 +1,387 @@
+#define SMMUV3_STREAM_TABLE_ENTRY_EATS_NOT_SUPPORTED                       0     // ATS not supported
+#define SMMUV3_STREAM_TABLE_ENTRY_S2VMID                                   1     // Pick non-zero value
+#define SMMUV3_STREAM_TABLE_ENTRY_S2TG_4KB                                 0     // Granule size 4KB
+#define SMMUV3_STREAM_TABLE_ENTRY_S2AA64                                   1     // AA64
+#define SMMUV3_STREAM_TABLE_ENTRY_S2TTB_OFFSET                             4     // S2TTB offset >> 4
+#define SMMUV3_STREAM_TABLE_ENTRY_S2PTW                                    1     // S2PTW bit
+#define SMMUV3_STREAM_TABLE_ENTRY_S2SL0                                    2     // Encoded level of page table walk, 2 -> 4-level page table starting from 0
+#define SMMUV3_STREAM_TABLE_ENTRY_OUTPUT_ADDRESS_MAX                       48    // 48 bit output address width max
+#define SMMUV3_STREAM_TABLE_ENTRY_S2RS_RECORD_FAULTS                       2     // Record faults
+#define SMMUV3_STREAM_TABLE_ENTRY_SHCFG_INCOMING_SHAREABILITY              1     // Incoming shareability attribute
+#define SMMUV3_STREAM_TABLE_ENTRY_SHCFG_INNER_SHAREABLE                    3     // Inner shareable
+#define SMMUV3_STREAM_TABLE_ENTRY_MTCFG                                    1     // MTCFG bit
+#define SMMUV3_STREAM_TABLE_ENTRY_MEMATTR_INNER_OUTTER_WRITEBACK_CACHED    0xF   // Inner+Outer write-back cached
+#define SMMUV3_STREAM_TABLE_ENTRY_VALID                                    1     // Entry is valid
+
+//
+// SMMUV3 Configuration bit definitions
+//
+#define SMMUV3_STR_TAB_BASE_CFG_FMT_LINEAR  0                // Linear Stream Table format
+#define SMMUV3_STR_TAB_BASE_ADDR_OFFSET     6                // Stream Table base address offset
+#define SMMUV3_STR_TAB_BASE_CMDQ_OFFSET     5                // Command queue base address offset
+#define SMMUV3_STR_TAB_BASE_EVENTQ_OFFSET   5                // Event queue base address offset
+#define SMMUV3_CR2_E2H                      0                // E2H bit 0
+#define SMMUV3_CR2_REC_INV_SID              1                // Record C_BAD_STREAMID for invalid input streams
+#define SMMUV3_CR2_PTM                      1                // PTM bit
+#define SMMUV3_CR0_EVENTQ_EN                1                // Event queue enable
+#define SMMUV3_CR0_CMDQ_EN                  1                // Command queue enable
+#define SMMUV3_CR0_SMMU_EN                  1                // SMMU enable
+#define SMMUV3_CR0_EVENTQ_EN                1                // Event queue enable
+#define SMMUV3_CR0_CMDQ_EN                  1                // Command queue enable
+#define SMMUV3_CR0_PRIQ_EN_DISABLED         0                // Disable PRI queue
+#define SMMUV3_CR0_VMW_DISABLED             0                // Disable VMID wildcard matching
+#define SMMUV3_CR0_ATS_CHK_DISABLE          1                // Disable bypass for ATS translated traffic
+
+typedef UINT64 PageTableEntry;

```

Comment:
```suggestion
typedef UINT64 PAGE_TABLE_ENTRY;

```


EDK 2 coding standard:  https://tianocore-docs.github.io/edk2-CCodingStandardsSpecification/release-2.20/4_naming_conventions/43_identifiers.html#table-4-reserved-keywords


4.3.5.1 Use all capital letters for both #define and typedef declarations.

Reply:
done